deploy/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: nginx-gateway
+description: NGINX Kubernetes Gateway
+type: application
+version: 0.1.0
+appVersion: "0.4.0"
+home: https://github.com/nginxinc/nginx-kubernetes-gateway
+sources:
+  - https://github.com/nginxinc/nginx-kubernetes-gateway/tree/v0.5.0/deploy/helm-chart
```

Comment:
I think this should just be the source of the project itself, not the helm chart, right?

Reply:
It's ambiguous, but I read it as home -> source of project; sources -> actual source code of the generated chart tarball:

```
home: The URL of this projects home page (optional)
sources:
  - A list of URLs to source code for this project (optional)
```
Otherwise it's the same link twice. Also, I think this is what was generated by the initial Helm command 😅